src/js/12-accessibility-fixes.js

```diff
@@ -0,0 +1,34 @@
+;(() => {
+  'use strict'
+
+  const addBlankAltTextsToTipIcons = () => {
+    const tipSVGs = document.querySelectorAll('.tip-svg')
+    tipSVGs.forEach((svg) => {
+      svg.setAttribute('alt', '')
+    })
+  }
+
+  const onDetailsExpand = (e) => {
+    const elem = e.target
+    setDetailsAriaExpanded(elem)
+  }
+
+  const setDetailsAriaExpanded = (elem) => {
+    if (elem.hasAttribute('open')) {
+      elem.setAttribute('aria-expanded', true)
+      return
+    }
+    elem.setAttribute('aria-expanded', false)
+  }
```

Comment:
this can be rewritten as:

```
const setDetailsAriaExpanded = (elem) => {
  elem.setAttribute('aria-expanded', elem.hasAttribute('open'));
}
```

Reply:
Well I feel silly now. Yes, that is a great call.